js/index.js

```diff
@@ -1 +1,131 @@
 // Your code goes here
+let headerImage = document.querySelector("#fun-bus");
+let containerDiv = document.querySelector(".container.home");
+let inputEl = document.querySelector("#input");
+let btnEl = document.querySelectorAll(".btn");
+let aEl = document.querySelectorAll("a");
+let counter = 0;
+
+function rotate(el, time, degreeFrom) {
+  TweenMax.to(el, time, {
+    transform: "rotateY("+degreeFrom+"deg)"
+  })
+}
+
+function createEl(msg) {
+  // console.log('counter',counter);
+  let check = document.querySelectorAll('.temp-message');
+  if(check.length === 0) {
+    let newDiv = document.createElement('div');
+  if (counter >= 40) {
+    newDiv.textContent = "I'm watching you...";
+    counter = 0;
+  } else {
+    newDiv.textContent = msg;
+  }
+    newDiv.setAttribute("class","temp-message");
+    newDiv.setAttribute("style","background: white; border-radius: 10px; padding: 20px; box-shadow: 2px 2px 4px black; position: fixed; top: 100px; left: 100px;")
+    containerDiv.appendChild(newDiv);
+    setTimeout(fadeToDelete, 1000);
+  }
+}
+
+function fadeToDelete() {
+  TweenMax.to(document.querySelectorAll('.temp-message'), 0.75, {
+    opacity: 0,
+    onComplete: deleteOver
+  })
+}
+
+function deleteOver() {
+  let temp = document.querySelectorAll('.temp-message');
+  temp[0].remove();
+}
+
+
+headerImage.addEventListener("mouseover", () => {
+  counter++;
+  rotate(headerImage, 1, 180);
+})
+
+headerImage.addEventListener("mouseout", () => {
+  counter++;
+  rotate(headerImage, 1, 0);
+})
+
+window.addEventListener("keydown", () => {
+  counter++;
+  createEl("you pressed a key!");
```

Comment:
these createEl functions are not working for me either, and i can't seem to figure out why. the only one that comes up is the "page has fully loaded!" message.

Reply:
yeah, it uses the Tween as part of the function. since it's having trouble loading the GSAP it won't work. if you want to take a look at it I can show it to you on my computer where it's working find.